src/params.cpp

```diff
@@ -168,9 +168,9 @@ void newton_param_input (const char *filenm,
             if (val!=1 || strcmp(buffer,"=")!=0) {
                 status = ERROR_INPUT_PAR; break;
             }
-            val = fscanf(fp,"%d",&ibuff);
+            val = fscanf(fp,"%lf",&dbuff);
             if (val!=1) { status = ERROR_INPUT_PAR; break; }
-            inparam->max_cells = ibuff;
+            inparam->max_cells = int(dbuff+0.5);
```

Comment:
Because when you ou 2E6 it would read it as 2. So you had to write 2000000 if you wanted that. I fixed it, you can write a float and it cast it as an int.